PORTAL/jobs/_utils.py

```diff
@@ -4479,21 +4631,12 @@ def _extract_kwargs(cls, data, optional, filename):
             raise ValueError(f'missing required data (fields: {missing})')
         return (kwargs, extra) if kwargs else (extra, None)
 
-    def refresh(self, resfile):
-        """Reload from the given file."""
-        fresh = self.load(resfile)
-        # This isn't the best way to go, but it's simple.
-        vars(self).clear()
-        self.__init__(**vars(fresh))
-        return fresh
```

Comment:
+1

Presumably this is used anywhere and not worth keeping.

Reply:
Yeah, that's what it looks like.